templates/taxi/car_detail.html

```diff
@@ -9,4 +9,7 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+
+  <h3><a href="{% url 'taxi:car-update' pk=car.id %}">Update ✏</a> |
+  <a style="color: lightcoral" href="{% url 'taxi:car-delete' pk=car.id %}">Delete 🗑</a></h3>
```

Comment:
I think it's better to add `class="btn btn-primary"` to make the elements look like a button